.travis.yml

```diff
@@ -41,6 +41,10 @@ install:
       fi
   - pip install -U tox twine wheel codecov
 
+addons:
+  hosts:
+    - xn--pxaix.localhost
```

Comment:
For me, locally, it works (on xbuntu 16.10). I don't know for others to be honest.
Happy to get your feedback on this.
This is what Travis reported without this: https://travis-ci.org/scrapy/scrapy/jobs/231217365